README.md

```diff
@@ -44,12 +44,12 @@ The Flume 1.x guide and FAQ are available here:
 
 ## Contact us!
 
-* Mailing lists: https://cwiki.apache.org/confluence/display/FLUME/Mailing+Lists
```

Comment:
The file you added comments to is a markdown file, so the syntax you have used is not the right one.
Also, the comments would have very limited additional value.
I would recommend closing this pull request.